api/models.py

```diff
@@ -74,4 +74,4 @@ class Node(ModelId):
     name: str
     revision: Revision
     parent: Optional[PyObjectId]
-    status: Optional[bool] = None
+    status: Optional[bool] = False
```

Comment:
We need to keep the default value as `None` to indicate there's no result yet.  So I think we'll need to fix the issue differently, maybe using an enum with string values.

Reply:
Resolved issue with `status`.